sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -98,8 +98,12 @@ class SparkSqlAstBuilder(conf: SQLConf) extends AstBuilder {
       ctx.identifier != null &&
       ctx.identifier.getText.toLowerCase == "noscan") {
       AnalyzeTableCommand(visitTableIdentifier(ctx.tableIdentifier).toString)
-    } else {
+    } else if (ctx.identifierSeq() == null) {
```

Comment:
Since this PR changes the Parser, please update the comment of this function to reflect the latest changes. 

In addition, please add the test cases in `DDLCommandSuite` for verifying the Parser's behaviors